composer.json

```
@@ -33,7 +33,7 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "psr/container": "^1.0|^2.0",
-        "yiisoft/arrays": "^2.1",
+        "yiisoft/arrays": "^2.1 || ^3.0",
```

```suggestion
        "yiisoft/arrays": "^2.1|^3.0",
```

See https://github.com/yiisoft/docs/blob/master/000-packages.md#composerjson